github

```diff
@@ -0,0 +1,3 @@
+f = loadfile('github.rom/programs/github')
+setfenv(f, getfenv())
+f(...)
```

Comment:
Can and should this be one-lined to `setfenv(loadfile('github.rom/programs/github'), getfenv())(...)`?